core/domain/opportunity_services.py

```diff
@@ -239,8 +239,15 @@ def update_opportunity_with_updated_exploration(exp_id):
 
     exploration_opportunity_summary.validate()
 
-    _save_multi_exploration_opportunity_summary(
-        [exploration_opportunity_summary])
+    # After all available content has been translated, delete the opportunity.
```

Comment:
IIRC, there's an issue around line 221 i.e, `complete_translation_languages` is not a part of the model and that's causing this problem i.e, the opportunity isn't getting updated! Can you please check and fix it?

Reply:
Ah, yes. Then I think we can persist `complete_translation_languages` (and update it here), and maybe when we fetch translation opportunities, only return those opportunities for which the language code is not in `complete_translation_languages`.

Reply:
Yeah, that sounds good! (Make sure to remove the `complete_translation_languages` field from the model on line 221.)